quickstarts/aws/amazon-database-migration-service/config.yml

```diff
@@ -0,0 +1,33 @@
+slug: amazon-database-migration-service
+description: |-
+  ## What is Amazon Database Migration Service?
+  AWS Database Migration Service (AWS DMS) is a web service you can use to migrate data from your database that is on-premises, on an Amazon Relational Database Service (Amazon RDS) DB instance, or in a database on an Amazon Elastic Compute Cloud (Amazon EC2) instance to a database on an AWS service.
+  ### Get started!
+  Start monitoring Amazon Database Migration Service by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-database-migration-service-monitoring-integration/) to learn more about New Relic monitoring for Amazon Database Migration Service.
+summary: |-
+  Monitor Amazon Database Migration Service by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Amazon Database Migration Service
+documentation:
+  - name: Amazon Database Migration Service installation docs
+    description: |
+      Monitor Amazon Database Migration Service by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-database-migration-service-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - Database Migration Service
```

Comment:
```suggestion

```

Reply:
@rahulbasu-nr This Quickstart is for Database Migration Service, I don't understand why do we need to remove this keyword, could you please help me with this.

Reply:
The same keyword exists in line no. 27. So I suggested to remove line no. 25.